assets/css/startertheme.css

```diff
@@ -0,0 +1,82 @@
+html,
+body {
+	margin: 0;
+	color: #333;
+	background: #eee;
+	font: sans-serif;
+}
+
+body {
+	padding: 20px;
+}
+
+.nav-link {
+	color: black;
+}
+
+li > a {
+    position: relative;
+    color: #000;
+    text-decoration: none;
+}
+
+li > a:hover {
+    color: #000;
+}
+
+
+li > a::before {
+  content: "";
```

Comment:
Can you fix the indenting for this file, it should be 2 spaces everywhere not 4.